Gemfile.lock

```diff
@@ -47,6 +47,14 @@ GEM
     haml (5.2.0)
       temple (>= 0.8.0)
       tilt
+    haml-lint (0.999.999)
```

Comment:
😕

Reply:
Yeah kinda weird, but that's what gets added when running `bundle add haml-lint`.